apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/phasing/PhasingPollServiceTest.java

```diff
@@ -56,20 +68,15 @@
 import org.junit.jupiter.api.parallel.Execution;
 import org.junit.jupiter.api.parallel.ExecutionMode;
 
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.function.Consumer;
-import javax.inject.Inject;
-
 @EnableWeld
 @Execution(ExecutionMode.CONCURRENT)
 public class PhasingPollServiceTest {
 
     @RegisterExtension
-    DbExtension extension = new DbExtension();
+    DbExtension extension = new DbExtension(DbTestData.getDbFileProperties(createPath("targetDb").toAbsolutePath().toString()));
```

Comment:
File db is not required here

Reply:
No, it's required here. Otherwise two test methods fail with error:

A file path that is implicitly relative to the current working directory is not allowed in the database URL "jdbc:h2:file:**null**/apl-blockchain-shard-0000001;;MV_STORE=FALSE;CACHE_SIZE=262144". Use an absolute path, ~/name, ./name, or the baseDir setting instead. [90011-196]